vendor/bat-native-ledger/src/bat/ledger/internal/endpoint/payment/payment_util.cc

```diff
@@ -19,16 +21,23 @@ std::string GetServerUrl(const std::string& path) {
   DCHECK(!path.empty());
 
   std::string url;
-  switch (ledger::_environment) {
-    case type::Environment::DEVELOPMENT:
-      url = kDevelopment;
-      break;
-    case type::Environment::STAGING:
-      url = kStaging;
-      break;
-    case type::Environment::PRODUCTION:
-      url = kProduction;
-      break;
+  const base::CommandLine& command_line =
```

Comment:
why are you using the command line here, you should be following the existing patterns and passing the url from rewards to ledger cc @tmancey

Reply:
sorry, this was already covered by @zenparsing above, I am suggesting the same thing

Reply:
resolved: https://github.com/brave/brave-core/pull/7557/files#diff-9709798eaebc36bee441ed094f6ed204130fc6dbfad28a2cc87f10c26772265f